lib/Support/Support.cpp

```diff
@@ -77,4 +78,26 @@ std::string escapeDottyString(const std::string &str) {
 }
 
 void report(const char *msg) { errs() << msg; }
+
+const std::string vformat(const char *format, ...) {
```

Comment:
can you add a test :)

Reply:
Absolutely. The only reason I haven't added it yet is that we don't have any tests for functions defined in `Support.h`.

Reply:
Done. Added a test.